nixos/modules/services/networking/networkmanager.nix

```diff
@@ -510,6 +537,16 @@ in {
         ];
       }
 
+      # if cfg.enableBundledFccUnlockScripts is set, populate
+      # networking.networkmanager.fccUnlockScripts with the values from
+      # pkgs.modemmanager.passthru.fccUnlockScripts.
```

Comment:
We don't have networking.networkmanager.package?

Reply:
The unlock scripts are provided by modemmanager, not NetworkManager, even though it's in the "networkmanager" config namespace.

Moving this into a separate namespace is out of scope for this PR.